src/hooks/useCurrentFaultTree.tsx

```diff
@@ -1,19 +1,19 @@
 import * as React from "react";
 import { createContext, useContext, useEffect, useState } from "react";
-
 import { FaultTree } from "@models/faultTreeModel";
 import * as faultTreeService from "@services/faultTreeService";
 import { axiosSource } from "@services/utils/axiosUtils";
 import { ChildrenProps } from "@utils/hookUtils";
 import { SnackbarType, useSnackbar } from "@hooks/useSnackbar";
 
-type faultTreeContextType = [FaultTree, () => void];
+
+type faultTreeContextType = [FaultTree, () => void, number];
 
 export const faultTreeContext = createContext<faultTreeContextType>(null!);
 
 export const useCurrentFaultTree = () => {
-  const [faultTree, refreshTree] = useContext(faultTreeContext);
-  return [faultTree, refreshTree] as const;
+  const [faultTree, refreshTree, rootReqProb] = useContext(faultTreeContext);
```

Comment:
@LaChope @kostobog let's discuss this pattern of creating hooks today

+

```
import * as React from "react";
import { createContext, useContext, useEffect, useState } from "react";
```

Reply:
Addressed with #246